subsys/net/lib/download_client/src/http_download_client.c

```diff
@@ -428,10 +545,6 @@ int download_client_start(struct download_client *client, char *file,
 
 	LOG_INF("Downloading: %s [%u]", client->file, client->progress);
```

Comment:
```suggestion
	LOG_INF("Downloading: %s [%u]", log_strdup(client->file), client->progress);

Reply:
Another thing that should be changed if some changes are going in is LOG_INF("Downloading: %s [%u]", client->file, client->progress);

Reply:
@lemrey ?

Reply:
Done, thanks.